test/Microsoft.ML.TestFramework/DataPipe/TestDataPipe.cs

```diff
@@ -1047,6 +1049,18 @@ public void SavePipeLda()
                     "loader=Text{col=F1V:Num:0-2}",
                     "xf=Lda{col={name=Result src=F1V numtopic=3 alphasum=3 ns=3 reset=+ t=1} summary=+}",
                 }, forceDense: true);
+
+            // topic summary text file saved inside the model.zip file.
+            string name = TestName + ".zip";
+            string modelPath = GetOutputPath("SavePipe", name);
+            using (var file = Env.OpenInputFile(modelPath))
+            using (var strm = file.OpenReadStream())
+            using (var zip = new ZipArchive(strm, ZipArchiveMode.Read))
+            {
```

Comment:
_(appended to code to allow for threaded comments)_

Can you please also add a test to test your PR on ML.NET's API instead of MAML?
I think modifying a test such as [InspectLdaModelParameters ](https://github.com/dotnet/machinelearning/blob/45a16dc14e631daa8e3ff441c7792a642ffdd312/test/Microsoft.ML.Functional.Tests/IntrospectiveTraining.cs#L169-L200) would be enough. Thanks.!

Reply:
Great idea, I will test both MAML and from transformer. Actually test from MAML and check summary was how TLC originally test topic summary.

---
In reply to: [446344620](https://github.com/dotnet/machinelearning/pull/5260#discussion_r446344620) [](ancestors = 446344620)